Selenium_Basics/EpamNavigationTests2.cs

```diff
@@ -0,0 +1,109 @@
+using OpenQA.Selenium;
+using OpenQA.Selenium.Chrome;
+
+namespace Selenium_Basics;
+
+public class EpamNavigationTests2
+{
+    private IWebDriver driver { get; set; }
+
+    [SetUp]
+    public void SetupBrowser()
+    {
+        var options = new ChromeOptions();
+        options.BinaryLocation = "/Applications/Google Chrome.app/Contents/MacOS/Google Chrome";
+        driver = new ChromeDriver(options);
+    }
+    
+    [TearDown]
+    public void TearDown()
+    {
+        driver.Quit();
+    }
+    
+    [Test]
+    public void CheckThatListOfCountriesContainsAmerEMEAandAPAC()
+    {
+        var epamUrl = "https://www.epam.com/";
+
+        driver.Manage().Window.Maximize();
+        driver.Navigate().GoToUrl(epamUrl);
+        driver.FindElement(By.XPath("//*[text()='Careers']//ancestor::*[@class='top-navigation__item-text']")).Click();
+        driver.FindElement(By.XPath("(//*[name()='use'])[10]")).Click();
+        bool Americas = driver.FindElement(By.XPath("//a[@data-item='0']")).Displayed;
+        bool EMEA = driver.FindElement(By.XPath("//a[@data-item='1']")).Displayed;
+        bool APAC = driver.FindElement(By.XPath("//a[@data-item='2']")).Displayed;
+        
+        Assert.True(Americas, "not displayed");
+        Assert.True(EMEA, "not displayed");
+        Assert.True(APAC, "not displayed");
+    }
+    
+    [Test]
+    public void CheckThatUrlToSearchContainsMyText()
+    {
+        var epamUrl = "https://www.epam.com/";
+        var searchUrl = "https://www.epam.com/search?q=Automation";
+        var keyWord = "Automation";
+            
+        driver.Manage().Window.Maximize();
+        driver.Navigate().GoToUrl(epamUrl);
+        driver.FindElement(By.XPath("//div[@class='header-search-ui header-search-ui-23 header__control']")).Click();
```

Comment:
//div[@class='header-search-ui header-search-ui-23 header__control']
this locator contains version in it - 23. With new version your test will fail. Plase review locator and use contains